homeassistant/components/moscow_yandex_transport/sensor.py

```diff
@@ -0,0 +1,143 @@
+# -*- coding: utf-8 -*-
+'''
+Service for obtaining information about closer bus from Transport Yandex Service
+@author: rishatik92@gmail.com
```

Comment:
We don't include the author here. We have the codeowner key in the manifest as the closest thing.